schema/me/artwork_inquiries.js

```diff
@@ -0,0 +1,43 @@
+import gravity from '../../lib/loaders/gravity';
+import Artwork from '../artwork/index';
+import { pageable, getPagingParameters } from 'relay-cursor-paging';
+import {
+  connectionDefinitions,
+  connectionFromArraySlice,
+} from 'graphql-relay';
+import {
+  GraphQLObjectType,
+  GraphQLID,
+  GraphQLNonNull,
+} from 'graphql';
+
+export const ArtworkInquiryType = new GraphQLObjectType({
+  name: 'ArtworkInquiry',
+  fields: () => ({
+    id: {
+      type: GraphQLID,
+    },
+    artwork: {
+      type: new GraphQLNonNull(Artwork.type),
+      resolve: ({ inquireable }) => inquireable,
+    },
+  }),
+});
+
+export default {
+  type: connectionDefinitions({ nodeType: ArtworkInquiryType }).connectionType,
+  args: pageable({}),
+  description: 'A list of the current user’s inquiry requests',
+  resolve: (root, options, request, { rootValue: { accessToken } }) => {
+    if (!accessToken) return null;
+    const { limit: size, offset } = getPagingParameters(options);
+    const gravityArgs = { size, offset, inquireable_type: 'artwork', total_count: true };
```

Comment:
one tiny point to keep in mind is you're requesting the count with _every_ request, whereas an optimization (not for now), could be to just request it on page 1 and keep it around.

There's probably not really a reason to always request it (esp. as it may change during paging), not to mention it's not a free request for the API.

anyways food for thought